en/Plugins/Publish.md

```diff
@@ -1,43 +1,49 @@
-The Publish plugin lets you publish notes from your vault onto your site, all within Obsidian.
+The Publish plugin lets you publish notes from your vault onto your site, all from within the Obsidian app.
 
-For more information about the Obsidian Publish service, what's included, how it works, and privacy go to the [[Obsidian Publish]] add-on service page.
+For more information about the Obsidian Publish™ service, what's included, how it works, and privacy, see the [[Obsidian Publish]] add-on service.
 
 ### Enable the plugin
 
-First, enable the Publish plugin. After that, you'll see the Publish icon appear in your [[Ribbon]], which looks like a paper plane.
+From within Obsidian > Settings > Plugin, enable the Publish plugin. After that, a Publish icon (which looks like a paper plane) will appear in the [[Ribbon]] pane.
 
 Once enabled, you can also set a hotkey for this option, or use the command palette to access it.
 
 ### Set up your site
 
-When purchasing Obsidian Publish, you'll be asked how many sites to purchase. You can then log in under Settings -> Account, so that the Publish plugin is aware of the number of remote sites you have.
+When purchasing Obsidian Publish, you'll be asked how many sites to purchase. Once you login under Settings > Account, the Publish plugin will display the number of publish sites you've purchased.
 
-If you have available slots, after opening Publish, you'll see an option to set up a site by entering a unique site ID. Your site will be available at `https://publish.obisidian.md/{site-id}`.
+If you have available sites, after opening Publish you'll see an option to set up a site by entering a unique site ID. After publishing, your site will be available at `https://publish.obisidian.md/{site-id}`.
 
 ### Site options
 
-After creating a site, you might want to give it a name, set a theme, and toggle the components (graph view, table of contents, etc.). You can do that by clicking on the gear icon near your site name, once you've created it.
+After creating a site, you have the option to give it a name, set a theme, and toggle various components (e.g. graph view, table of contents, etc.). You can do that by clicking on the gear icon near your site name at the top of the Publish plugin.
```

Comment:
Don't think we should use "e.g." together with "etc" 🤔 

Not 100% sure but Google showed me this: https://meryl.net/ie-etc-and-eg/#:~:text=Rule%20%231%3A%20Don't,apple%2C%20oranges%2C%20etc.%E2%80%9D&text=Since%20et%20means%20and%2C%20avoid%20using%20and%20with%20etc.

Reply:
Great catch!!! Totally agree.